### ISSUE TITLE
Support for sending optional git data to coveralls

### ISSUE THREAD

**Issue by jmiranda** [agorapulse/grails-coveralls#5]
Thank you for this plugin. I was wondering if you had plans to support sending git data to the coveralls API (see git argument https://coveralls.zendesk.com/hc/en-us/articles/201350799-API-Reference). If so, I can try to put together a pull request to handle that feature.

**Comment by benorama** [comment]
Hi,

I've got no plan to work on this plugin lately.
We are going to migrate to Grails3 :).

Please go ahead if you want to add this feature.

**Comment by jmiranda** [comment]
I've attached an example showing a coveralls build with git info is included when using circleci (still haven't figured out how to include the commit message yet) ![telmediq sp grails coveralls test coverage history statistics 2](https://cloud.githubusercontent.com/assets/1162935/11817893/ae90ddc8-a31c-11e5-9236-604bcf0f9e8b.png). I'll create a pull request once I ensure everything looks ok.

Used the node-coveralls plugin as an example
https://github.com/nickmerwin/node-coveralls/blob/master/lib/getOptions.js

@benorama Curious whether git data is send by default with TravisCI? If not, I can add that as well.

**Comment by benorama** [comment]
The call is to Coveralls is currently done here: https://github.com/agorapulse/grails-coveralls/blob/master/src/groovy/grails/plugin/coveralls/api/JobsAPI.groovy

So I guess you could add more info to the report json payload.

**Comment by jmiranda** [comment]
Exactly. That's what I ended up doing. I've only got it working with circleci but supporting other CI services is pretty trivial. 
https://github.com/jmiranda/grails-coveralls/commit/47cb5f2d5a5af8458a3124e1be37328496d58c59

**Comment by lthur** [comment]
I have created this PR to add this functionality!
I do really need this feature too, so that was the reason I've create the PR.
https://github.com/agorapulse/grails-coveralls/pull/8

Any comments, let me know!

Thank you!
